docs/components/cursor.md

```diff
@@ -80,6 +80,7 @@ AFRAME.registerComponent('cursor-listener', {
 | downEvents  | Array of additional events on the entity to *listen* to for triggering `mousedown` (e.g., `triggerdown` for vive-controls).  | []                               |
 | fuse        | Whether cursor is fuse-based.                                                                                              | false on desktop, true on mobile |
 | fuseTimeout | How long to wait (in milliseconds) before triggering a fuse-based click event.                                             | 1500                             |
+| mode     | Camera (gaze) or mouse controlled. | gaze
```

Comment:
A boolean `mouseMode` property properly represents the mutual exclusiveness, that the entire component is flipped to a mouse mode or it's not. "Modes" usually mean an entire configuration, not "additional functionality".

"Gaze mode" does not make sense as the cursor can be used with controllers as well, and in the component code, "gaze mode" doesn't trigger any logic. It's only when setting to mouse mode does any different code run.

How I could see this component being used (although I'm not too amped on including a mouse mode in the first place):

```
<a-scene>
  <a-mixin id="raycaster" raycaster="objects: .clickable"></a-mixin>

  <a-entity mixin="raycaster" cursor="mouseMode: true"></a-entity>

  <a-camera>
    <a-entity mixin="raycaster" cursor></a-entity>
  </a-camera>

  <a-entity mixin="raycaster" laser-controls</a-entity>
</a-scene>
```